ts/screens/wallet/AddCardScreen.tsx

```diff
@@ -142,11 +141,22 @@ const primaryButtonPropsFromState = (
   const card = getCreditCardFromState(state);
 
   return card.fold<BlockButtonProps>(
-    {
-      ...baseButtonProps,
-      disabled: true,
-      accessibilityRole: "button"
-    },
+    (e: string | undefined) =>
+      e === undefined
+        ? {
+            ...baseButtonProps,
+            disabled: true,
+            accessibilityRole: "button"
+          }
+        : {
+            ...baseButtonProps,
+            disabled: true,
+            accessibilityRole: "button",
+            accessibilityLabel: I18n.t(
+              "wallet.dummyCard.accessibility.button.error",
+              { field: e }
+            )
+          },
```

Comment:
```suggestion
      ({
      ...baseButtonProps,
      disabled: true,
      accessibilityRole: "button",
      accessibilityLabel: e
        ? I18n.t("wallet.dummyCard.accessibility.button.error", {
            field: e
          })
        : undefined
    }),
```
Could it be a semplification?

Reply:
Yes, updated in [81f0680](https://github.com/pagopa/io-app/pull/3020/commits/81f0680471b539b9647c525173149968640f985e)